shared/src/business/entities/EntityConstants.ts

```diff
@@ -1723,3 +1723,8 @@ export const STATUS_REPORT_ORDER_OPTIONS = {
     restored: 'restoredToGeneralDocket',
   },
 };
+
+export const TROUBLESHOOTING_INFO = {
+  APP_SUPPORT_EMAIL: 'dawson.support@ustaxcourt.gov',
+  FILE_UPLOAD_TROUBLESHOOTING_LINK: '', // TODO 10001
```

Comment:
@jtdevos Once we know where the file troubleshooting link will be, I will put it here :)

Reply:
Thanks Chris - will follow-up when Tenille and I settle on a name.

Reply:
Looks like we're gonna go with `https://ustaxcourt.gov/dawson_faqs_case_management.html#FileUpload`. Note that this page hasn't been updated yet, but it will be in place prior to deployment.